### DISCUSSION TITLE
Welcome to pdf.js Discussions!

### DISCUSSION THREAD

**Discussion by calixteman** [mozilla/pdf.js#17234] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by baiyiting** [comment]
Hello, I have a requirement that I don't know how to implement: I need to filter out the outline that does not contain these matches on the basis of the matches I searched for after entering the keyword, I don't know how to do it, can you provide an idea?

**Comment by umerfarooq4455** [comment]
how to run this project and impliment in the react tsx code
